sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -358,6 +327,15 @@ private static PodSpec convertPod(
       builder.hostVolumes(hostVolumeSpecs);
     }
 
+    if (!rawPod.getExternalVolumes().isEmpty()) {
+      Collection<HostVolumeSpec> externalVolumeSpecs = new ArrayList<>();
+      externalVolumeSpecs.addAll(rawPod.getExternalVolumes().values().stream()
+              .map(v -> convertHostVolume(v))
```

Comment:
```suggestion
              .map(v -> convertExternalVolume(v))
```

I think you meant `convertExternalVolume` here instead of `convertHostVolume`